test-generator/src/generator_help.cr

```diff
@@ -91,6 +91,22 @@ class GeneratorHelp
     input.tr("-", "_").camelcase
   end
 
+  def function(input : JSON::Any, arg_types : Array(String))
```

Comment:
Can you add some documentation for this function? It wasn't clear to me what this was for until I looked at how it was used in the template.